tests/test_integration.py

```diff
@@ -527,10 +520,31 @@ def get_existing():
         self.assertEqual(existing.count, count + 1)
 
     def test_update_aggregate_size_existing_merged(self):
-        existing = models.ScheduleABySize.query.filter_by(
-            size=0,
-            cycle=2016,
-        ).first()
+        def get_existing():
+            return models.ScheduleABySize.query.filter_by(
+                size=0,
+                cycle=2016,
+            ).order_by(
+                models.ScheduleABySize.committee_id,
+            ).first()
```

Comment:
Out of curiosity, why the method definition inside of another method?  If I understand correctly, this block of code is defined as a method so that it executes and therefore does the database lookup at the time of the variable assignment down below so that we're guaranteed to retrieve the latest updates in the database at that point in the test, correct?  If so, is this just a matter of scoping?  I've always been a bit fuzzy on Python's flexibility in allowing this kind of design pattern.

Reply:
It is just a matter of scoping. It allows us to have multiple methods named `get_existing`, each specific to the test within which it is defined. e.g. in `test_update_aggregate_asize_existing` and `test_update_aggregate_size_existing_merged`.

Reply:
Got it, thank you!